shared-libs/rules-engine/test/integration.spec.js

```diff
@@ -146,6 +146,8 @@ describe('Rules Engine Integration Tests', () => {
     // the task is 5 days old when it is discovered
     const NOW = MS_IN_DAY * 5;
     sinon.useFakeTimers(NOW);
+    // Reset the "calculatedDate"
+    await rulesEngine.updateEmissionsFor(['patient']);
```

Comment:
I think this might actually nullify the test. The patient should get marked as "dirty" by time passing (the useFakeTimers call simulates time passing).

Reply:
I changed my timezone to Phoenix Arizona and checked the exact error here. 
The errors that I got were because I got extra documents written in this test, namely targets. This means that, in the setup (beforeEach) where3 we actually do an `updateEmissionsFor`, we were on a different reporting date than in the test. Which makes sense, because the ts is 1 and your TS is with -. 

I fixed the tests locally by changing the starting date to be `MS_IN_DAY`, so we make sure that _everyone_ is in the same reporting period and targets don't get recalculated. 
As a result, I had to change more timestamps across the file, but no logic-altering changes were needed. 

To be very clear, this is the line that I changed: 
https://github.com/medic/cht-core/blob/0d4c275b088a92edb5b971d4f40aafcde3f93d42/shared-libs/rules-engine/test/integration.spec.js#L110

Reply:
Ah, yes this is exactly the kind of thing I was worried about.  Your `MS_IN_DAY` fix seems to work perfectly!  Thanks!